### PR TITLE
YouTube Subproject Genesis

### PR DESCRIPTION
**Pull request by chris-short**
The Stream Team

Includes fixes to links to youtube-guideline.md that would break as a result of this commit.

Signed-off-by: Chris Short <chris@chrisshort.net>

Never created a subproject before so flog me as necessary.

### CODE REVIEW

**Comment by mrbobbytables** [comment]
Thanks for kicking it off @chris-short :)

Just an FYI - All the sig README's are autogenerated -- The links and information are made by updating 
[sigs.yaml](https://git.k8s.io/community/sigs.yaml) with the subproject information then running `make generate` or `make generate-dockerized`. That will auto-update the README and any other possible files with the appropriate information.

**Comment by castrojo** [comment]
Send a message to the contrib ex list proposing this as a subproject so we can give people a 72 hour window to give feedback, I'll lgtm when you sort out the README business. Overall +1 to this idea, breaking this section into subprojects will make scaling out easier.

**Comment by parispittman** [comment]
this should be a subproject of communication imo

**Comment by mrbobbytables** [comment]
> this should be a subproject of communication imo

Just for clarification, under "[community-management](https://github.com/kubernetes/community/blob/master/sigs.yaml#L1038)"?
The project that owns the communications directory?

**Comment by parispittman** [comment]
my bad - yes, in community management; wherever we house the communication guidelines, moderation, etc. unless this will have a special repo in k-sigs but i don't think this needs to get that heavy.

**Comment by castrojo** [comment]
OK so what's the next action here, looks like chris just needs to cut out the subproject definition in the readme and everything else in the directory inherits the owner's from up the directory tree right?

**Comment by parispittman** [comment]
you should create an OWNERs file and have the streaming team in there. i would rename this directory to youtube-team like we are doing with the others that are like this. create the role, too, as youtube-streamer.md like the release team has with scope, responsibilities, etc.

File: communication/youtube/OWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
```
Comment (parispittman) [inline communication/youtube/OWNERS]: can you put me in as a reviewer?

**Comment by parispittman** [comment]
@chris-short if you can add me in as a reviewer, that's all we need to finish this up. thanks for your help with this!!

File: sig-contributor-experience/README.md [hunk 2]
```diff
@@ -79,6 +79,9 @@ The following subprojects are owned by sig-contributor-experience:
     - https://raw.githubusercontent.com/kubernetes/community/master/events/OWNERS
   - Meetings:
     - Contributor Summit strategy, content and planning: [Mondays at 9:00 PT (Pacific Time)]() (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:00&tz=PT%20%28Pacific%20Time%29).
+- **youtube**
```
Comment (mrbobbytables) [inline sig-contributor-experience/README.md]: This section can be removed, it will live under the communication subproject 👍

**Comment by chris-short** [comment]
I'm absolutely terrible at GH notifications

File: communication/youtube/OWNERS (hunk 1 above)
Comment (chris-short) [reply communication/youtube/OWNERS]:
```suggestion
reviewers:
parispittman
```

Comment (chris-short) [reply communication/youtube/OWNERS]:
```suggestion
reviewers:
parispittman
```

File: sig-contributor-experience/README.md (hunk 2 above)
Comment (chris-short) [reply sig-contributor-experience/README.md]:
```suggestion
```

File: sig-contributor-experience/README.md [hunk 3]
```diff
@@ -79,6 +79,9 @@ The following subprojects are owned by sig-contributor-experience:
...
+  - Owners:
```
Comment (chris-short) [inline sig-contributor-experience/README.md]:
```suggestion
```

File: sig-contributor-experience/README.md [hunk 4]
```diff
@@ -79,6 +79,9 @@ The following subprojects are owned by sig-contributor-experience:
...
+    - https://raw.githubusercontent.com/kubernetes/communication/master/youtube/OWNERS
```
Comment (chris-short) [inline sig-contributor-experience/README.md]:
```suggestion
```

**Pull request closed without merging**